packages/headless/src/controllers/commerce/search/did-you-mean/headless-did-you-mean.ssr.ts

```diff
@@ -0,0 +1,13 @@
+import {SSRCommerceEngine} from '../../../../app/commerce-engine/commerce-engine.ssr';
```

Comment:
I created a separate Jira to add this one to the sample.

https://coveord.atlassian.net/browse/KIT-3463

It'll be very easy to do, but we first need to have a search page setup.